### DISCUSSION TITLE
v3.3.0 Update Early Testing - Tabbed Portals and Better Multi-Monitor Support

### DISCUSSION THREAD

**Discussion by Ross-Patterson** [Ross-Patterson/Portals-Desktop-Organization#305] [Announcements]
v3.3.0.4 is available for early testing and I would appreciate any support in the testing effort.

**Before installing, take a backup of: 
_C:\Users\username\AppData\Local\Portals_
So you don't lose your settings if something goes wrong.**

Download Here:
[Downloads Portals v3.3.0.4](https://downloads.portals-app.com/build-previews%2Fv3.3.0.4%2Fportals_installer_v3-3-0-4.exe?alt=media)

Thanks!

**Comment by CraftyCNC** [comment]
3.3.0.4 working well for me. Thank you for   Hide on double click fix.

**Reply by dhf-esi** [reply]
_Thank you for Hide on double click fix._ **Seconded**.

**Comment by dhf-esi** [comment]
When I installed this version the previous version had some portals currently showing subfolders.  These subfolders became the root folders for the portal with the new version.
Unexpected, slightly annoying, thought the folder navigation was broken at first.

**Reply by Ross-Patterson** [reply]
Good to know, will try and see if I can replicate this and fix it.

**Comment by GnitLores** [comment]
When _Auto Expand on Hover_ is enabled, clicking a tab sometimes causes the portal to collapse.

![HJwG7WvXb3](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/11030459/28f97ae8-b298-49cc-ba60-30b8285f9409)

This seems kind of similar to the existing issue where a portal sometimes collapses when an icon is right clicked.

![2f8QxrpUow](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/11030459/5fb9e48b-7f7f-410f-b49b-e4dcfe2e861f)

**Reply by Ross-Patterson** [reply]
Thanks, will take a look into it

**Comment by PhoenixR1s1ng** [comment]
I love seeing this implemented! The tabs work well and they make the portals much more useful.
I have noticed that the tabs can sometimes get pretty big depending on the folder name. A tab "width " modifier could be pretty useful here.
Great work!

**Comment by sherpa25** [comment]
Hi, I have 3.3.0.9.  Is it possible to have Tabs inside a main Portal (ie. Portal, and inside are Tabs 1, 2, etc.) and the name Portal-name?  Currently it shows the name of the Tab selected, even if the Portal is closed.
